src/Access/Authentication.cpp

```diff
@@ -83,11 +84,55 @@ bool Authentication::isCorrectPassword(const String & password_, const String &
         case LDAP_SERVER:
         {
             auto ldap_server_params = external_authenticators.getLDAPServerParams(server_name);
```

Comment:
I need to adjust the user name and password, so I have to copy it. I will see if this can be worked around by explicitly providing those two values, but I suspect it will be ugly. Copying couple of strings shouldn't be a problem here, this is not a tight place.